Gemfile

```diff
@@ -1,20 +1,17 @@
 source 'https://rubygems.org'
 
-gem 'rake'
-gem 'asciidoctor', '1.5.6.1'
+gem 'rake', '13.0.6'
+gem 'asciidoctor', '2.0.16'
 
-gem 'json'
-gem 'awesome_print'
+gem 'json', '2.6.1'
+gem 'awesome_print', '1.9.2'
 
-gem 'asciidoctor-epub3', :git => 'https://github.com/asciidoctor/asciidoctor-epub3'
-gem 'asciidoctor-pdf', '1.5.0.alpha.16'
-gem 'asciidoctor-pdf-cjk', '~> 0.1.3'
```

Comment:
Unless it has changed lately, you have to keep `asciidoctor-pdf-cjk` in order to be able to build a PDF with CJK chars. This requirement pulls other requirements on `asciidoctor-pdf` and `asciidoctor`

Reply:
Did you try to compile a PDF book with these settings?